### DISCUSSION TITLE
2023年5月以降のイベント出展について

### DISCUSSION THREAD

**Discussion by potappo** [mozilla-japan/translation#693] [General]
関連ディスカッション：https://github.com/mozilla-japan/translation/discussions/678

次のイベント出展（ブース参加）または開催（ホスト開催）について議論するスレッドです。

以下は、OSC 2023 東京春のディスカッションから転載および加筆しています。

今年の次の OSC リアル開催は名古屋で 5/28（日）です。

https://event.ospn.jp/osc2023-nagoya/

日曜ということもあり、私がメインで動くことはできないですが、誰かメインで出展したい方がいるなら、当日手伝いに行けるように調整します。@hmatrjp さんとかどうでしょう？
なお、出展申込締切は 4/21（金）です。今月のミートアップの前日になります。

さらにその次は、6/24 (土）に北海道（札幌）での開催があります。こちらには私は法事の時期と重なってしまうため参加できません。もしどなたか出展したい方がいるなら、準備段階で支援できますといった程度です。

なお、OSC 東京秋は会場探し中（11 月？） とのこと。こちらには都合悪くなければメインで参加の予定です。
5/20 追記：5/10 時点での情報ですが、10/21(土）に大田区の PiO を仮押さえしたという情報が事務局から出ています。まだ確定ではないので、最新の情報は、OSPN Slack で確認してください。この日で確定した場合、ミートアップ開催日をずらして参加の予定です。

あとは、翻訳ミートアップをリアル会場開催するかですね。リアル会場開催するなら、会場参加者にノベルティグッズを配っても良いかと思います。こちらは、会場の第 1 候補となる WebDINO Japan さんとの相談になるので、日程は白紙の状態です。

他にブース出展できるイベントを知っている方がいれば、情報提供をお願いします。

例えば、私単独で執筆する気はないですが、技術同人誌を執筆した上で、技術同人系イベントに出てはどうか？という話も以前にありました。

私としては、土日祝開催のイベントを想定しています。お盆や年末などに開催するイベントの場合は、早めに連絡をくだされば、調整できるかもしれません（ただしそれも土日に近い曜日前提となります）。

**Comment by dynamis** [comment]
皆様 OSC 出展お疲れ様でした。今回年度末対応で燃えてて何も出来ない間に終わってて申し訳ない。

> あとは、翻訳ミートアップをリアル会場開催するかですね。リアル会場開催するなら、会場参加者にノベルティグッズを配っても良いかと思います。こちらは、会場の第 1 候補となる WebDINO Japan さんとの相談になるので、日程は白紙の状態です。

弊社オフィスを貸すことは場所としては問題ないハズですが、当日に開けて対応する人を誰にするか(私もたまに出来ると思うが毎回は難しい)がポイントです。
リアル開催をしたいというお話であれば検討・調整しますが、皆様のご希望的にはどうでしょうかね。

**Reply by gurezo** [reply]
@dynamis cc @potappo @hmatrjp 
5~7月開催くらいで、調整さんを立ててオフィスの日程を確認するという案はどうでしょうか？

**Reply by potappo** [reply]
@dynamis 
ありがとうございます。
対応される方は、Isaku さん（@fuzz139）が以前対応していらっしゃったので、引き続き Isaku さんか、Isaku さんに当時をヒアリングして引き継げる方であれば問題ないと思います。

@gurezo 
各月の日程としては、これまで通り、20 日に一番近い土曜日というルールを採用したいですが、チューターできる人の都合も合えば、WebDINO Japan のオフィスの都合を考慮してずらすでも問題ないと思います。候補日としてはずらっと土曜を出してみる感じでしょうか。

**Comment by hmatrjp** [comment]
外出やら腰痛やら重なって、お返事遅れました。
OSC名古屋 5/28 のリアル会場ブース出展は、近辺の予定を見てみると、前日が先約で潰れるため厳しいです。
OSC東京2023春のように、ノベルティグッズを数日前に発送することはできると思います。

当日見に行くだけなら行けるかなあと考えていたところでした。ここもセミナーは別開催でしたね。

**Comment by gurezo** [comment]
OSC Tokyo の秋の方が良いかもしれませんね。
自分も５月から作業環境が変わるので、難しいです。

**Comment by potappo** [comment]
濱崎さん、ぐんまーさん、了解です。
今のところ、次のイベントは、時期は今後調整の翻訳ミートアップリアル会場開催、OSC 東京秋といったところですね。
